src/recommendations/RecommendationsPage.jsx

```diff
@@ -59,7 +77,7 @@ const RecommendationsPage = () => {
     setTimeout(() => { handleSkipRecommendationPage(); }, LINK_TIMEOUT);
   };
 
-  if (!registrationResponse) {
+  if (!redirectUrl && !showRecommendations) {
```

Comment:
same 
> this would disturb the authn mfe registration flow

Reply:
How?
This wouldn't disturb the authn mfe registration flow because if there is no `redirectUrl` and the user is not coming onboarding experience then It should be redirected to the dashboard, and if redirectUrl exist user will able to see the recommendations

Reply:
you are right.